src/util/util_env.cpp

```diff
@@ -34,7 +34,20 @@ namespace dxvk::env {
       ? fullPath.substr(n + 1)
       : fullPath;
   }
-  
+#else
+  std::string getExeName() {
+    char exePath[MAX_PATH + 1];
+
+    DWORD len = ::GetModuleFileNameA(NULL, exePath, MAX_PATH);
+
+    std::string fullPath(exePath);
+    auto n = fullPath.find_last_of('\\');
+
+    return (n != std::string::npos)
+      ? fullPath.substr(n + 1)
+      : fullPath;
+  }
+#endif  
```

Comment:
Does the old function not work with wine? If that's the case, it might even be better to use this one for everything. Having two almost identical functions around is poor style.

Reply:
Those functions are redefined because they don't work with winebuild. While other do work (`getEnvVar`, function `getTempDirectory` - don't but currently unused),  . Have no idea why.

This expected to not work with unicode filenames, your example produces `?????_dxgi.log` for renamed `dxgi-factory.exe`. But it's not big problem, as for me.

Reply:
What this basically means is that file paths containing anything non-Latin1 won't work. This is unacceptable. I might take a look at this later but this needs to be resolved.